network/bitcoin_transaction.ksy

```diff
@@ -62,88 +62,90 @@ types:
         contents: [0xff, 0xff, 0xff, 0xff]
         doc: |
           Magic number indicating the end of the current input.
+    types:
+      script_signature:
+        seq:
+          - id: len_sig_stack
+            type: u1
+          - id: der_sig
+            type: der_signature
+            doc: |
+              DER-encoded ECDSA signature.
+            doc-ref: |
+              https://en.wikipedia.org/wiki/X.690#DER_encoding
+              https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm
+          - id: sig_type
+            type: u1
+            enum: sighash_type
+            doc: |
+              Type of signature.
+          - id: len_pubkey_stack
+            type: u1
+          - id: pubkey
+            type: public_key
+            doc: |
+              Public key (bitcoin address of the recipient).
+        types:
+          der_signature:
```

Comment:
there is already a spec for ASN.1

Reply:
Right. I will have a look when I get a chance. Thanks!